oak-commons/src/main/java/org/apache/jackrabbit/oak/commons/collections/CollectionUtils.java

```diff
@@ -81,4 +81,17 @@ public static <T> Set<T> toSet(final Iterator<T> iterator) {
         return result;
     }
 
+    /**
+     * Convert a vararg list of items to a set.  The returning set is mutable and supports all optional operations.
+     * @param elements elements to convert
+     * @return the set
+     * @param <T> the type of the elements
+     */
+    public static <T> Set<T> toSet(@SuppressWarnings("unchecked") final T... elements) {
+        final Set<T> result = new HashSet<>();
```

Comment:
Should we specify the size for the HashSet?

Reply:
Well, it'll be a NPE. I actually do not like the additional checks; they provide zero additional information, unless they come with a message.